src/update-feeds-loops/update-feeds-loops.test.ts

```diff
@@ -443,6 +444,7 @@ describe(updateFeedsLoopsModule.processBatch.name, () => {
     jest.spyOn(logger, 'info');
 
     // Skip actions other than generating signed api urls.
+    jest.spyOn(gasPriceModule, 'fetchAndStoreGasPrice').mockImplementation();
```

Comment:
Mhh, why do these need to be mocked now?

Reply:
Yes it looks redundant but I wanted to add them because `processBatch` includes `fetchAndStoreGasPrice`. Should I remove them?

Reply:
Nah, leave them. I was just surprised why the test was passing, but I guess either Ethers made a noop call or silently did nothing 🤷

Reply:
Probably because `fetchAndStoreGasPrice` handles error by returning null when it fails. However, this spy on `fetchAndStoreGasPrice` is good to have because I encountered [this](https://github.com/ethers-io/ethers.js/blob/ad5f1c5fc7b73cfb20d9012b669e9dcb9122d244/src.ts/providers/provider-jsonrpc.ts#L767) error when I am playing with tests even though I couldn't produce it now.